state/txindex/indexer_service.go

```diff
@@ -4,26 +4,34 @@ import (
 	"context"
 
 	"github.com/tendermint/tendermint/libs/service"
-
+	"github.com/tendermint/tendermint/state/indexer"
 	"github.com/tendermint/tendermint/types"
 )
 
+// XXX/TODO: These types should be moved to the indexer package.
```

Comment:
want to open a issue?

Reply:
Sure, but I'm going to tackle it as soon as this PR is merged.